packages/server/src/app.module.ts

```diff
@@ -33,7 +33,7 @@ import { OAuth2Client } from "google-auth-library";
         TypeOrmModule.forFeature([Carpool, Driver, Passenger, User]),
         PassportModule.register({ defaultStrategy: "jwt" }),
         JwtModule.register({
-            secretOrPrivateKey: authConfig.secret,
+            secret: authConfig.secret,
```

Comment:
I was getting an error in the console saying `secretOrPrivateKey` was deprecated and that we should use `secret` instead.